### ISSUE TITLE
[video play]: show seeders/leechers _per file_

### ISSUE THREAD

**Issue by haschimoto** [zer0net/_ppload-videorama-tickets#11]
idea for code is here:
https://gitlab.com/kickasstorrents/KAT/tree/master/utils/lib-seeders

here is the thread to discuss:
http://127.0.0.1:43110/Talk.ZeroNetwork.bit/?Topic:3_13Mt8T2256Uz4GHk9B4XvKqyz37wPgA26q

**Comment by dnelband** [comment]
currently immposible, herllo Zero Net has special permissions, thats why it can show peers. tamas wrote that it would be ready in the next week or two

**Comment by haschimoto** [comment]
Okay, lets wait for the official solution then.

**Comment by dnelband** [comment]
tamas sent me an email with instructions how to implement, it seems like a good idea to implement it now because it will solve the cross origin problem as well (e.g hopefully no need for different firefox solution) apart from giving us access to seeders / leechers & other mergerSites features

**Comment by dnelband** [comment]
Videos now have peer information (only those who belong to a channel with merged:videorama)

**Comment by haschimoto** [comment]
Is that true file based seeders number?
Since videos are independent, optional files, it could be totally different
and not related to the number of connected peers of the main site...

**Comment by dnelband** [comment]
True, this is an estimate. Im not sure its possible with optional files but ill ask tamas as well as check other solutions
